tests/unit/test_nat64.cpp

```diff
@@ -31,6 +31,7 @@
 #include "test_platform.h"
 #include "test_util.hpp"
 
+#include "inttypes.h"
```

Comment:
```suggestion
#include <inttypes.h>
```
Maybe change the line below to `<string.h`>?

Reply:
Done.